via/static/scripts/video_player/components/icons/Pause.tsx

```diff
@@ -14,6 +14,7 @@ export default function PauseIcon(props: PauseIconProps) {
       height="16"
       fill="none"
       data-component="PauseIcon"
+      viewBox="0 0 16 16"
```

Comment:
This prevents the SVG from getting clipped when rendered at smaller sizes.

Reply:
This is an auto-generated component. Can you update the source SVG and regenerate instead.